packages/eslint-config-sheriff/tsdown.config.ts

```diff
@@ -0,0 +1,14 @@
+import { defineConfig } from 'tsdown';
+
+export default defineConfig({
+  entry: ['./src/index.ts'],
+  format: 'esm',
+  target: 'node20',
+  platform: 'node',
+  clean: true,
+  sourcemap: false,
+  skipNodeModulesBundle: true,
+  publint: { strict: true },
+  noExternal: ['@sherifforg/constants', '@sherifforg/types'],
+  dts: { isolatedDeclarations: true },
```

Comment:
This should be inferred, but there's a bug right now: https://github.com/rolldown/tsdown/issues/126